### PR TITLE
Document scanning

### PR DESCRIPTION
**Pull request by hermannakos**

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/DocumentScanner/src/main/java/com/zynksoftware/documentscanner/ui/imagecrop/ImageCropFragment.kt [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        }
+    }
+
+    private fun onError(error: DocumentScannerErrorModel) {
+        if (isAdded) {
+            getScanActivity().onError(error)
+        }
+    }
+
+    private fun onConfirmButtonClicked() {
```
Comment (hermannakos) [inline apps/DocumentScanner/src/main/java/com/zynksoftware/documentscanner/ui/imagecrop/ImageCropFragment.kt:118]: Here instread of going to the ImageProcessingFragment return the cropped image and handle image processing in our code.

File: apps/student/src/main/java/com/instructure/student/util/AppManager.kt [hunk 2]
```diff
@@ -17,36 +17,12 @@
...
-import com.instructure.pandautils.utils.ColorKeeper
-import com.instructure.student.BuildConfig
-import com.instructure.student.R
-import com.instructure.student.flutterChannels.FlutterComm
-import com.instructure.student.service.StudentPageViewService
 import com.instructure.student.tasks.StudentLogoutTask
-import com.pspdfkit.PSPDFKit
-import com.pspdfkit.exceptions.InvalidPSPDFKitLicenseException
-import com.pspdfkit.exceptions.PSPDFKitInitializationFailedException
+import com.zynksoftware.documentscanner.ui.DocumentScanner
```
Comment (tamaskozmer) [inline apps/student/src/main/java/com/instructure/student/util/AppManager.kt:24]: This import is unused.

File: apps/teacher/build.gradle [hunk 3]
```diff
@@ -202,6 +202,7 @@ dependencies {
     implementation project(path: ':rceditor')
     implementation project(path: ':annotations')
     implementation project(path: ':interactions')
+    api project(path: ':DocumentScanner')
```
Comment (tamaskozmer) [inline apps/teacher/build.gradle:205]: Do we need this in the teacher app?

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

**Review by tamaskozmer — APPROVED** [review]

**Pull request merged**